faqs/galaxy/workflows_import_from_dockstore.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Import workflows from DockStore
+area: workflows
+box_type: tip
+layout: faq
+contributors: [nekrut]
+---
+
+[Dockstore](https://dockstore.org/) is a free and open source platform for sharing reusable and scalable analytical tools and workflows.
+
+-----
+
+The following short video walks you through this uncomplicated procedure:
+
+<p align="center"><iframe width="560" height="315" src="https://www.youtube.com/embed/K2wFrSLFpa0?si=mcaCXKNx3JNbLnBv" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe></p>
```

Comment:
```suggestion
<p align="center"><iframe width="560" height="315" src="https://www.youtube-no-cookie.com/embed/K2wFrSLFpa0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe></p>
```

- please remove tracking IDs.
- please use the `youtube-no-cookie` domain to respect the privacy of our users.